File: SoloLearn Codes/Gotham City/gotham_city.cpp
#include <iostream>
using namespace std;

int main() {
    int c;
    cin >> c;
    if (c < 5) cout << "I got this!";
    else if (c >= 5 && c <= 10) cout << "Help me Batman";
    else cout << "Good Luck out there!";
    return 0;
}

File: SoloLearn Codes/Gotham City/GothamCity.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int crim = Convert.ToInt32(Console.ReadLine());
            if (crim < 5) Console.WriteLine("I got this!");
            else if (crim >= 5 && crim <= 10) Console.WriteLine("Help me Batman");
            else Console.WriteLine("Good Luck out there!");
        }
    }
}
